quipucords/api/source/serializer.py

```diff
@@ -232,24 +206,27 @@ def update(self, instance, validated_data):
         source_type = instance.source_type
         credentials = validated_data.pop('credentials', None)
         hosts_data = validated_data.pop('hosts', None)
+        if hosts_data:
+            hosts_list = json.loads(hosts_data)
```

Comment:
Same comment as above.

Reply:
check not needed here since it is done in validate_hosts